.mdignore

```diff
@@ -0,0 +1 @@
+vendor
```

Comment:
Since `vendor` is handled in the `markdownlint` target, this isn’t necessary any more (but worth keeping as an example?).